### ISSUE TITLE
Project still live?

### ISSUE THREAD

**Issue by markaudacity** [Kroc/Paperclip#1]
Is this project still in development?

**Comment by Kroc** [comment]
No, sorry. C++ is too much for me as someone who doesn't even know C, and given the hundreds of hours I would have to put into it to make it merely functional for a piece of software possibly nobody else would ever use on hardware that is dying out and can't be replace :P However, the code that's here provides a complete starting framework for writing apps on EPOC32 because I sussed out all the complex build process stuff
